apps/src/templates/manageStudents/ManageCodeReviewGroups.jsx

```diff
@@ -1,31 +1,137 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
 import PropTypes from 'prop-types';
 import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
 import i18n from '@cdo/locale';
+import Spinner from '@cdo/apps/code-studio/pd/components/spinner';
 import StylizedBaseDialog from '@cdo/apps/componentLibrary/StylizedBaseDialog';
-import CodeReviewGroupsLoader from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsLoader';
 import CodeReviewGroupsStatusToggle from '../codeReviewGroups/CodeReviewGroupsStatusToggle';
+import CodeReviewGroupsManager from '@cdo/apps/templates/codeReviewGroups/CodeReviewGroupsManager';
 
 const DIALOG_WIDTH = 1000;
+const STATUS_MESSAGE_TIME_MS = 5000;
```

Comment:
why do we only want to show the message for a limited time? 5 seconds is not very long, and if it disappears it might imply there was success when there was not.

Reply:
Are you referring to the error message? The success message also only shows for 5 seconds -- I just thought you'd click submit and see it fail (or succeed). Mostly just wanted the success message to disappear so if you started doing more dragging and dropping you wouldn't think that those changes were saved. Maybe I could remove the success message once you make any subsequent change to groups, and keep the error message until you hit submit again?

Reply:
I think having the success message disappear once you make changes would work. We should show the error message until you retry, does it currently disappear or does it persist?

Reply:
It also disappears, I can leave it until you retry